_posts/2019-11-20-agile-reporting.md

```diff
@@ -0,0 +1,254 @@
+For this, I used the velocity of the team to generate a burnup chart so that they could see by what date we would arrive at the end. 
+That's called forecasting.  
+
+<span class="image left"><img class="p-image" alt="Timeline" src="{{ '/img/2019-11-20-agile-reporting/reporting-timeline.png' | prepend: site.baseurl }}"></span>
+
+But there's immediately a problem there, velocity is a **lagging indicator**.  
+It shows long-term patterns and can confirm such patterns - for example every December, the velocity would drop due to team members going on holidays.
+It is difficult to use velocity as a planning tool since you can't see in the future. 
+And it's based on estimations of a backlog, which are subject to change.
+
+So the end date I had to report to management would typically move forward each meeting.  
+Needless to say, they weren't happy with that. 
+Every single time, I had to explain the fundamentals of Scrum, the backlog, refinements, story splitting, etc...
+
+## Throughput Forecasting
+
+A useful tool I learned with Doc Norton's talk is the [throughput forecasting tool](https://github.com/FocusedObjective/FocusedObjective.Resources/raw/master/Spreadsheets/Throughput%20Forecaster.xlsx){:target="_blank" rel="noopener noreferrer"}.  
+It's an Excel file where you can enter the following data:
+* The min/max backlog/feature size.
+* The % of story splitting (hard to calculate, not required).
+* The start date (typically today).
+* The duration of your sprints.
+* The min/max velocity.
+* With an estimate.
+* Or with actual data from your project.
+
+The tool then runs 500 simulations where it randomly picks numbers between the minimum and maximum you provided. 
+Those simulations are then used to give you a probability chance of reaching a certain date.
+
+The past few meetings I've started to use this tool and it helped me to provide a better, probable end date of the features.
+
+Let's take an example. 
+In our backlog, we have 3 features waiting, a total of 150 story points. 
+Now, you have to estimate how many sprints it would take to finish and by what date.
+
+Assume we start 01-01-2019 and work with sprints of 2 weeks. 
+The past 5 velocities were 60, 55, 40, 45, 51, averaging in 50. 3 sprints, great!  
+Estimated end date = 01-01-2019 + 6 weeks = 12-02-2019
+Now, let's put that in the tool.
+
+<img alt="Throughput Tool Input" src="{{ '/img/2019-11-20-agile-reporting/reporting-throughput-input.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+The result:
+
+<img alt="Throughput Tool Result" src="{{ '/img/2019-11-20-agile-reporting/reporting-throughput-results.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+So there's a 55% probability that the features will be done in 3 sprints.  
+However, there's a 100% probability that it will be done in 4 sprints. And that's just for 3 features worth of 150 story points.
```

Comment:
```suggestion
However, there's a 100% probability that it will be done in four sprints.
And that's just for three features worth of 150 story points.
```